pkg/config/validation/validation.go

```diff
@@ -855,15 +855,8 @@ func validateOutlierDetection(outlier *networking.OutlierDetection) (errs error)
 	if outlier.BaseEjectionTime != nil {
 		errs = appendErrors(errs, ValidateDurationGogo(outlier.BaseEjectionTime))
 	}
-	if outlier.ConsecutiveErrors < 0 {
-		errs = appendErrors(errs, fmt.Errorf("outlier detection consecutive errors cannot be negative"))
-	}
-	if outlier.Consecutive_5XxErrors != nil || outlier.ConsecutiveGatewayErrors != nil {
-		// ConsecutiveErrors is deprecated for Consecutive_5XxErrors and
-		// ConsecutiveGatewayErrors; they should not be set at the same time.
-		if outlier.ConsecutiveErrors > 0 {
-			errs = appendErrors(errs, fmt.Errorf("consecutive_errors should not be set with consecutive_5xx_errors or consecutive_gateway_errors"))
-		}
+	if outlier.ConsecutiveErrors != 0 {
```

Comment:
Not sure if this should be reserved for some time or just be removed and ignore it silently.

Reply:
I think we should just ignore especially when we are removing the functionality in pilot. We can log a warning during deprecation period, but once it is removed - we should not evaluate that field.

Reply:
OK, i will just print a warning here